openapi3/codegen-templates/api_client.mustache

```diff
@@ -163,7 +163,8 @@ function Invoke-{{{apiNamePrefix}}}ApiClient {
                                         -ErrorAction Stop `
                                         -UseBasicParsing `
                                         -SkipCertificateCheck `
-                                        -UserAgent $OktaUserAgent
+                                        -UserAgent $OktaUserAgent `
+                                        -SkipHttpErrorCheck
```

Comment:
Since the `SkipHttpErrorCheck` param is supported by PS 7+, and the Okta PS module supports PS 6.2+, do you have any thoughts on how this should be handled in PS 6+?

Reply:
Ah, interesting one, I did look at Powershell 5.1, but didn't look at PS 6 vs 7 consideration apologies.

Off top of my head can think of three ways to handle it 
1) rewrite the code so it does try catches non-200 responses and in the catch it needs logic to separate 429, etc from other errors
2) adjust it with a case for PS6 vs 7 - probably using try catch in PS^ as theres no other way around that IWR  behvior and the new param in 7
3) Consider moving to PS7 - Im unsure of whether you can see the mix of PSVersions in use from PSGallery though - so that may not be an easy one either